tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -2727,7 +2727,6 @@ def unary_op_optional_output_with_fast_approx(
 transpose_cw = make_unary_op(partial(ttl.tensor.transpose, dim0=1, dim1=-1))
 eltwise_floor = make_unary_op_optional_output(ttnn.floor)
 # eltwise_ceil = make_unary_op_optional_output(ttnn.ceil)
-eltwise_ceil = make_unary_op(ttl.tensor.ceil)
```

Comment:
instead of deleting, use ttnn api 
eltwise_ceil = make_unary_op_optional_output(ttnn.ceil)